src/main/scala/org/broadinstitute/dsde/firecloud/core/ProfileClient.scala

```diff
@@ -340,14 +340,15 @@ class ProfileClientActor(requestContext: RequestContext) extends Actor with Fire
       case None => Put
     }
 
-    memberList flatMap { members => pipeline { rawlsRequestMethod(url, members) } }
-
-    // don't leak any sensitive data
-    RequestComplete(NoContent)
-  }.recover {
+    memberList flatMap { members => pipeline { rawlsRequestMethod(url, members) } } map { response =>
```

Comment:
While I do believe this is a slow call I have not seen it time out in production. And even if timeouts happen the background process goes on and the next sync will be a success. But that is only for this use case and it will only get worse.

So first, I think we should understand when this thing will timeout because it also will be a problem for changing permissions on a workspace. If we do need something asynchronous, we should establish a pattern for it that includes monitoring because it is not the only thing we will want it for.

I don't think that rawls needs to understand white lists. What we would implement is asynchronous group membership changes.

Note that I have already added the loggly alert for the rawls call. With or without this change we should still get the proper alerting.